pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -901,7 +901,24 @@ func FormatEnvName(name string) string {
 	if len(envName) > 63 {
 		envName = envName[len(envName)-63:]
 	}
-	return strings.Replace(envName, ".", "-", -1)
+
+	envName = strings.Replace(envName, ".", "-", -1)
+	envName = checkUsableNameEnvFile(envName, serviceName)
+	return envName
+}
+
+// checkUsableNameEnvFile checks and adjusts the environment file name to make it usable.
+// If the first character of envName is a hyphen "-", it is concatenated with nameService.
+// If the length of envName is greater than 63, it is truncated to 63 characters.
+// Returns the adjusted environment file name.
+func checkUsableNameEnvFile(envName string, serviceName string) string {
```

Comment:
since we're returning a string here, I suggest:
```suggestion
func getUsableNameEnvFile(envName string, serviceName string) string {
```

Reply:
applied